lib.nix

```diff
@@ -93,27 +68,4 @@ in lib // (rec {
     (name: env: f (env // { environment = name; }))
     environments;
 
-  runHaskell = name: hspkgs: deps: env: code: let
```

Comment:
this was being used by the yaml validation tests in hydra, are those still being ran?